src/ddmd/backend/gloop.c

```diff
@@ -794,9 +795,31 @@ void loopopt()
     /* one first, thus moving LIs out as far as possible.       */
 
     doflow = TRUE;                      /* do flow analysis             */
+
+    if (go.mfoptim & MFtime)
+    {
+        cmes("Starting loop unrolling\n");
+        for (loop *l = startloop; l; l = ln)
```

Comment:
If we're at naming `nl` for next loop is more obvious than `ln` which typically abbreviates line.